src/controllers/dungeonController.js

```diff
@@ -29,6 +30,16 @@ class DungeonController {
         res.status(err.message).json(errorResponse[err.message]);
       });
   }
+
+  postFinalWords(req, res) {
+    if (req.headers.heroid && req.headers.words) {
+      this.dungeonService.updateWords(req.headers.heroid, req.headers.words)

```

Comment:
as i requested on the frontend, pls moe the words to the payload from the headers

Reply:
solved